### PR TITLE
Minute

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?

We're adding the minute script to the `/uk`, `/us`, and `/video` fronts.
I have small concerns around perf - the script is 49k gzipped - but nothing to major.
## What is the value of this and can you measure success?

This will get us to a point that we can start look at how we're going to track - hence the `forceFeature` flag.
## Does this affect other platforms - Amp, Apps, etc?

No
## Screenshots

minute
## Request for comment

@akash1810 @gidsg @sndrs (<= re: perf).

### CODE REVIEW

**Comment by sndrs** [comment]
50k of js to fronts! we've been wringing our hands about react, i wouldn't say that was "nothing to major"! 

what does it do?

**Comment by jamesgorrie** [comment]
>  i wouldn't say that was "nothing to major"

That's why I thought it best to ask the professionals.
It does what you see in the screengrab - video teasers on video cards. 

Happy to run you through how it works - but there's more on [their website](http://minute.ly/).

**Comment by sndrs** [comment]
looking at the code, you load it optionally, and manually at that, is that correct?

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -485,4 +485,15 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  // Owner: Video / Multimedia team
+  val MinuteVideoTeaser = Switch(
+    SwitchGroup.Feature,
+    "minute-video-teaser",
+    "When ON, the user will be able to share comments",
+    owners = Seq(Owner.withGithub("guardian/multimedia")),
```
Comment (akash1810) [inline common/app/conf/switches/FeatureSwitches.scala]: might be worth creating a `Owner.withGithubGroup` helper?

Comment (jamesgorrie) [inline common/app/conf/switches/FeatureSwitches.scala]: Yip - thought of that - left it out of this PR.

**Comment by regiskuckaertz** [comment]
Is the teaser playing only when the card is hovered/active?

**Comment by paperboyo** [comment]
Does this use accessibility aid CSS class "flashing-image"? cc/@johnduffell

**Comment by jamesgorrie** [comment]
> Is the teaser playing only when the card is hovered/active?

It's not really teasing then if you're already +50% interested?

> Does this use accessibility aid CSS class "flashing-image"?

Good point. This is by no means the actual implementation - we are in the process of changing it to an A/B test - but will 100% make sure that's the case if we go through with it.

**Comment by regiskuckaertz** [comment]
> It's not really teasing then if you're already +50% interested?

Fair enough, although our eyes are able to detect subtle changes in our peripheral vision; and so this can be distracting and have a negative impact if people can't focus on what they're really interested in anymore. I really like the idea btw, and on the contrary the surprise effect of seeing the image move when you hover/tab the card could boost users' interest—especially if they are browsing.

**Comment by johnduffell** [comment]
merge conflicts and test failures 😞 anyone taking this one on now?

**Comment by akash1810** [comment]
> anyone taking this one on now?

@gidsg is though he's away until next week.

**Comment by NataliaLKB** [comment]
@gidsg @akash1810 should we close this until it is ready to reopen?

**Comment by gidsg** [comment]
Good shout @NataliaLKB will close and re-open when I have something ready to go.

**Pull request closed without merging**